service/src/utils/onchain-data.ts

```diff
@@ -120,7 +120,10 @@ export async function unwrapSarcophagus(web3Interface: Web3Interface, sarcoId: s
   }
 
   try {
-    await web3Interface.archaeologistFacet.unwrapSarcophagus(sarcoId, decryptedShard);
+    await web3Interface.archaeologistFacet.unwrapSarcophagus(
+      sarcoId,
+      ethers.utils.arrayify(decryptedShard) // TODO: Might need an extra `Buffer.from`. `unwrapSarcophagus` expects a solidity bytes type, `decryptedShard` is a hex string
+    );
```

Comment:
@DarksightKellar  -- im going to merge this in, but would be good to test this out this week